### DISCUSSION TITLE
Github copilot not showing suggestions

### DISCUSSION THREAD

**Discussion by Tamunonengiyeofori** [community/community#17595] [Copilot Conversations]
I code with python , I recently installed and setup Github copilot on Vscode but I am not seeing any suggestions on my python code. Can someone help me fix this ?

**Comment by davecheney** [comment]
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by Tamunonengiyeofori** [reply]
I think the Github copilot was loading or connecting to the Internet and that's what I couldn't get suggestions. It seems to be working now

**Comment by SankalpMangure** [comment]
I have already selected GitHub Copilot from view->Output but still, I am not getting any suggestion.

**Reply by davecheney** [reply]
Looking at your telemetry it looks like your employer, internet service provider, or government, may be intercepting your connection.
